pkg/interface/src/views/components/VirtualScroller.tsx

```diff
@@ -303,8 +303,10 @@ export default class VirtualScroller<T> extends Component<VirtualScrollerProps<T
     const ref = this.childRefs.get(this.savedIndex)!;
     //ref.scrollIntoView();
     const newScrollTop = this.window.scrollHeight - ref.offsetTop - this.savedDistance;
-    
+
+    this.window.style['-webkit-overflow-scrolling'] = 'auto';
     this.window.scrollTop = newScrollTop;
+    this.window.style['-webkit-overflow-scrolling'] = 'touch';
```

Comment:
What's this about?

Reply:
I remember patching this on Urbit.org once where touch scroll would feel really janky and unnatural. It wouldn’t glide.